packages/lesswrong/lib/scoring.ts

```diff
@@ -49,9 +72,55 @@ export const timeDecayExpr = () => {
   ]}
 }
 
-export const defaultScoreModifiers = () => {
+export const postScoreModifiers = () => {
   return [
     {$cond: {if: "$frontpageDate", then: FRONTPAGE_BONUS.get(), else: 0}},
     {$cond: {if: "$curatedDate", then: CURATED_BONUS.get(), else: 0}}
   ];
 };
+
+export const commentScoreModifiers = () => {
+  const {magnitude, durationHours} = getSubforumScoreBoost();
+
+  // (Date.now() - createdAt.getTime()) / 3600000
```

Comment:
Could you add a comment saying that this should give the same result as `getSubforumCommentBonus`